frontend/src/components/widgets/ColorPicker/ColorPicker.test.tsx

```diff
@@ -16,7 +16,7 @@
  */
 
 import React from "react"
-import { shallow } from "enzyme"
+import { shallow } from "lib/test_util"
```

Comment:
Mainly moving forward, we should use `lib/test_util` then `enzyme` for everything. Essentially if a component under it uses emotion themes it will error out.